vignettes/multiple-factors.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: "Issue of Multi-Factor Measurement Model"
```

Comment:
Issues or The Issue of (sounds grammarly more correct?)

Reply:
Thanks for catching this. Maybe I'll delete "Issue of"